lib/simctl.js

```diff
@@ -236,5 +236,13 @@ async function getDevices (forSdk:string = null):Object {
   return devices;
 }
 
+async function getScreenshot (udid:string):string {
+  let tempScreenshotPath = 'temp-screenshot-' + udid + '.png';
```

Comment:
I think it would be better to put it in a temp directory.

https://github.com/appium/appium-support/blob/master/lib/tempdir.js